packages/compass-components/src/components/generative-ai/generative-ai-input.tsx

```diff
@@ -246,7 +252,21 @@ function GenerativeAIInput({
             title={closeText}
             onClick={() => onClose()}
           >
-            <RobotSVG />
+            <AIGuideCue
+              open={isAggregationGeneratedFromQuery}
+              setOpen={() => {
+                /* noop because we control guide cue visibility through onResetIsAggregationGeneratedFromQuery */
+              }}
+              refEl={guideCueRef}
+              onResetIsAggregationGeneratedFromQuery={
+                onResetIsAggregationGeneratedFromQuery
+              }
+              title="Aggregation generated"
+              description="Your query requires stages from MongoDB's aggregation framework. Continue to work on it in our Agaredation Pipeline Builder"
```

Comment:
There's a typo we should fix here, how do you feel about this?
```suggestion
              description="The generated query requires use of MongoDB's aggregation framework. Open and work on it in the Aggregation Pipeline Builder"
```
or with the typo fixed:
```suggestion
              description="Your query requires stages from MongoDB's aggregation framework. Continue to work on it in our Aggregation Pipeline Builder"
```

Reply:
I think "Open and work on it in the Aggregation Pipeline Builder" is a bit confusing because this message shows up as a guide cue when we already navigated to the aggregation. But I will fix the typo, thx! did not notice it 👀

Reply:
Ah gotcha, I was thinking this was before the transition. Sounds good.